repl/tests/rexpect.rs

```diff
@@ -22,126 +19,137 @@ macro_rules! test {
     };
 }
 
+struct REPL {
+    session: PtySession,
+    prompt: &'static str,
+}
+
+impl REPL {
+    /// Defines the command, timeout, and prompt settings.
+    /// Wraps a rexpect::session::PtySession. expecting the prompt after launch.
+    fn new() -> Result<REPL> {
+        let command = "../target/debug/gluon -i";
+        let timeout: u64 = 10_000;
+        let prompt: &'static str = "> ";
+
+        let mut session = spawn(command, Some(timeout))?;
```

Comment:
Could use [spawn_command](https://docs.rs/rexpect/0.3.0/rexpect/session/fn.spawn_command.html) here.

```rust
spawn_command(Command::new(command).arg("-i").env("GLUON_PATH", ".."), Some(timeout))
```

Reply:
The .env method returns a &mut Command, while spawn_command only wants a Command, Moved the env part of the macro to new().